src/main/resources/not_enough_gamerules.mixins.json

```diff
@@ -20,7 +22,6 @@
     "entities.VillagerMixin",
     "entities.ZombieMixin",
     "entities.ZombieVillagerMixin",
-    "PistonMixin",
```

Comment:
Merci de classer par ordre alphabétique...

Reply:
C'est classé par ordre alphabétique
Les Majuscules sont classées indépendamment puis les minuscules
C'est automatique, ça fait ça a chaque fois que j'ajoute un Mixin, ça reclasse correctement

Reply:
Well it doesn't sort the way I want it sorted

Reply:
Okay so I'm going to put the weird sorting back